consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseMatchingServiceAPI.java

```diff
@@ -27,26 +27,22 @@
 
 public class DatabaseMatchingServiceAPI extends AbstractMatchingServiceAPI {
 
-    ConsentAPI consentAPI;
-    DataAccessRequestAPI dataAccessAPI;
-    DataSetAPI dsAPI;
-    private WebTarget target;
-    GenericType<ResponseMatchingObject> rmo = new GenericType<ResponseMatchingObject>(){};
-    protected Logger logger() {
-        return Logger.getLogger("DatabaseMatchingServiceAPI");
-    }
+    private ConsentAPI consentAPI;
+    private DataAccessRequestAPI dataAccessAPI;
+    private DataSetAPI dsAPI;
+    private WebTarget matchServiceTarget;
+    private GenericType<ResponseMatchingObject> rmo = new GenericType<ResponseMatchingObject>(){};
+    private Logger logger = Logger.getLogger("DatabaseMatchingServiceAPI");
 
     public static void initInstance(Client client, ServicesConfiguration config) {
         MatchAPIHolder.setInstance(new DatabaseMatchingServiceAPI(client, config, AbstractConsentAPI.getInstance(), AbstractDataAccessRequestAPI.getInstance(), AbstractDataSetAPI.getInstance()));
     }
 
-    protected DatabaseMatchingServiceAPI(Client client, ServicesConfiguration config, ConsentAPI consentAPI, DataAccessRequestAPI darAPI, DataSetAPI dsAPI){
+    DatabaseMatchingServiceAPI(Client client, ServicesConfiguration config, ConsentAPI consentAPI, DataAccessRequestAPI darAPI, DataSetAPI dsAPI){
```

Comment:
should this actually be private?

Reply:
It's used by a test which is in the same package space, so the default package-protected works here. Making it private would break the test.